### DISCUSSION TITLE
Add dark mode logo option

### DISCUSSION THREAD

**Discussion by declan-haojin** [jpanther/congo#532] [Ideas]
Sometimes the website may have different logos for light and dark mode. It would be great to have the option to specify a dark mode logo in the `params.toml` file. If no dark mode logo is specified, the default logo is used for both modes. If a dark mode logo is specified, the default logo is used for light mode and the dark mode logo is used for dark mode.

**Comment by jpanther** [comment]
This is now included in the next release.
